src/sections/Home/Proud-maintainers/proudMaintainers.style.js

```diff
@@ -103,6 +103,12 @@ const ProjectItemWrapper = styled.section`
           h4 {
             font-size: 0.7rem;
           }
+      }
+      @media screen and (max-width: 660px){
```

Comment:
What does this CSS do?

Reply:
I didn't add any styling to the blog, any styling you see there was auto-generated from the blog wrapper

Reply:
besides, there is no h4 tag in the index.mdx

Reply:
@mzoyinda Colud you please revert only your last commit, it's taking from others commit. We are good to merge then.

Reply:
These changes are not required, I assume.
Can you please revert it @mzoyinda .